train.py

```diff
@@ -30,3 +31,38 @@
 rnn_data_test = preparedata.rnn_inputs(
     dfs=normalized_test_dfs, features=input["features"], targets=input["targets"], window_length=input["window_length"]
 )
+
+# %% set variables for train and test
+
+# train
+train_x_rnns = rnn_data_train["x_rnns"]  # variables for plotting

```

Comment:
How are you splitting the training and validation data?

Reply:
I use tf.model.fit method to split train and validation. I set it randomly select 20% of my training data as the validation set.